compose-examples/prysm-only/docker-compose.yaml

```diff
@@ -91,11 +91,27 @@ services:
     user: "1000"
     depends_on:
       - prometheus
+      - renderer
     ports:
       - 127.0.0.1:3000:3000
     volumes:
       - ./config/grafana/provisioning-p:/etc/grafana/provisioning
       - ./data/grafana:/var/lib/grafana
+    environment:
+      GF_RENDERING_SERVER_URL: http://renderer:8082/render
+      GF_RENDERING_CALLBACK_URL: http://grafana:3000
+      GF_LOG_FILTERS: rendering:debug
+    <<: *logging
+
+  renderer:
+    image: grafana/grafana-image-renderer:2.0.0
+    restart: on-failure
+    hostname: renderer
+    ports:
+      - 127.0.0.1:8082:8082
+    environment:
+      ENABLE_METRICS: 'true'
```

Comment:
can you also connect this container to prometheus?

Reply:
now render is connected to prometheus